client-rest/src/app/transfer/auth-callback.component.ts

```diff
@@ -33,7 +34,12 @@ export class AuthCallbackComponent implements OnInit {
         if (Step.AUTHENTICATE_EXPORT === this.progressService.currentStep()) {
             // export auth step: received the export auth callback, generate the export auth data from the export token
             // and redirect to the import url. Use SSL as the token is sent cleartext.
-            this.transferService.generateAuthData({id: transferId, authToken: token, mode: "EXPORT"}).subscribe(
+            this.transferService.generateAuthData({
+              id: transferId,
+              authToken: token,
+              mode: "EXPORT",
+              callbackUrl: `${environment.apiBaseUrl}/callback/${this.progressService.exportService().toLowerCase()}`
```

Comment:
Do we expect to handle the service as part of the path in all clients?

Reply:
per offline discussion, some clients may choose to use the same callback, with no service in the path

the demo client always includes service/ in the path